src/main/java/com/bric/math/StringConverter.java

```diff
@@ -0,0 +1,45 @@
+package com.bric.math;
+
+import java.math.BigDecimal;
+
+public class StringConverter {
```

Comment:
I don't see a lot of value in this extra class, especially considering that as far as I understand Arrays.toString() could be used instead (at least in the one-dimensional case).